public/locales/ro/common.json

```diff
@@ -67,11 +67,8 @@
   "useful.links": "Linkuri utile",
   "legal.info": "Informații legale",
   "signup.resources.offer": "Resursa oferita:",
-  "signup.resources.offer.thankYouMessage": "Iți mulțumim pentru generozitate. Resursele pe care le-ai pus la dispoziție s-au înregistrat în sistem și imediat ce le vom aloca și oferi pentru a ajuta mai departe persoanele vulnerabile vom lua legătura cu tine.",
   "signup.resources.request": "Resursa solicitata:",
-  "signup.resources.request.thankYouMessage": "Îți mulțumim pentru mesajul transmis. Vom procesa cererea ta cât de repede posibil.",
```

Comment:
it's used in the `ThankYouMessage` component

![image](https://user-images.githubusercontent.com/5412091/157031948-bafa142b-9d05-4397-be7b-01e50337929d.png)